Lib.AspNetCore.ServerSentEvents/ServerSentEventsClientValidationArgs.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Lib.AspNetCore.ServerSentEvents
+{
+    /// <summary>
+    /// Provides data for the <see cref="IServerSentEventsService.ValidationHandler"/> event.
+    /// </summary>
+    public struct ServerSentEventsClientValidationArgs
```

Comment:
This is not used for validating client, but connection. Also this will not be used in context of an event, so it shouldn't be `*Args`. My porposal would be `ServerSentEventsConnectionValidationContext`